composer.json

```diff
@@ -1,7 +1,7 @@
 {
   "require": {
-    "krokedil/wp-api": "^1.0",
-    "krokedil/woocommerce": "dev-develop-free-trial-shipping as 1.3.1"
+    "krokedil/wp-api": "dev-develop",
```

Comment:
The `develop-free-trial-shipping` PR has not yet been merged into develop. If we change the branch, this might break subscription support in Klarna Payments.